packages/elements/src/number-field/index.ts

```diff
@@ -504,6 +559,16 @@ export class NumberField extends FormFieldElement {
     }
   }
 
+  /**
+   * Reset error state on input
+   * @returns {void}
+   */
+  private resetError(): void {
```

Comment:
it's not using anywhere

Reply:
Thank. This should not be there. Already removed.